### PR TITLE
Update the OpenSSL cookbook to v4

### PR DESCRIPTION
**Pull request by eherot**

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -16,6 +16,6 @@
 depends 'hostsfile'
 depends 'java'
 depends 'et_mesos', '~> 5.0'
-depends 'openssl', '~> 3.0'
+depends 'openssl', '>= 4.0'
```
Comment (jeffbyrnes) [inline metadata.rb]: In light of the clearly breaking changes to what that cookbook provides, wouldn’t a less permissive version pin make more sense?

Comment (eherot) [inline metadata.rb]: In light of [the PR I just merged](https://github.com/evertrue/et_mesos-cookbook/pull/6) I really don't think it's worth the trouble. Even the problem introduced in this case was fairly harmless other than preventing convergence.

Comment (jeffbyrnes) [inline metadata.rb]: Alright.

**Comment by jeffbyrnes** [comment]
🚢

**Pull request merged**